packages/dnsutils/build.sh

```diff
@@ -2,65 +2,29 @@ TERMUX_PKG_HOMEPAGE=https://www.isc.org/downloads/bind/
 TERMUX_PKG_DESCRIPTION="Clients provided with BIND"
 TERMUX_PKG_LICENSE="MPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=9.16.41
+TERMUX_PKG_VERSION=9.20.4
 TERMUX_PKG_SRCURL="https://ftp.isc.org/isc/bind9/${TERMUX_PKG_VERSION}/bind-${TERMUX_PKG_VERSION}.tar.xz"
-TERMUX_PKG_SHA256=71904366aa1e04e2075c79a8906b92af936e3bfa4d7e8df5fd964fcf9e94f45c
-TERMUX_PKG_DEPENDS="openssl, readline, resolv-conf, zlib, libuv"
+TERMUX_PKG_SHA256=3a8e1a05e00e3e9bc02bdffded7862faf7726ba76ba997f42ab487777bd8210b
+TERMUX_PKG_DEPENDS="openssl, readline, resolv-conf, zlib, libuv, liburcu, libcap, libandroid-glob, libnghttp2"
 TERMUX_PKG_BREAKS="dnsutils-dev"
 TERMUX_PKG_REPLACES="dnsutils-dev"
 TERMUX_PKG_BUILD_IN_SRC=true
 
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
---disable-linux-caps
---without-python
---with-ecdsa=no
---with-gost=no
---with-gssapi=no
---with-libjson=no
---with-libtool
---with-libxml2=no
---with-openssl=$TERMUX_PREFIX
---with-randomdev=/dev/random
---with-readline=-lreadline
---with-eddsa=no
 ax_cv_have_func_attribute_constructor=yes
 ax_cv_have_func_attribute_destructor=yes
+lt_cv_prog_compiler_pic_works=yes
+--disable-static
 "
 
 termux_step_pre_configure() {
-	export BUILD_AR=ar
-	export BUILD_CC=gcc
-	export BUILD_CFLAGS=
-	export BUILD_CPPFLAGS=
-	export BUILD_LDFLAGS=
-	export BUILD_RANLIB=
-
 	_RESOLV_CONF=$TERMUX_PREFIX/etc/resolv.conf
 	CFLAGS+=" $CPPFLAGS -DRESOLV_CONF=\\\"$_RESOLV_CONF\\\""
+	LDFLAGS+=" -landroid-glob"
 }
 
-termux_step_make() {
-	make -C lib/isc
-	make -C lib/dns
-	make -C lib/ns
-	make -C lib/isccc
-	make -C lib/isccfg
-	make -C lib/bind9
-	make -C lib/irs
-	make -C bin/dig
-	make -C bin/delv
-	make -C bin/nsupdate
-}
-
-termux_step_make_install() {
-	make -C lib/isc install
-	make -C lib/dns install
-	make -C lib/ns install
-	make -C lib/isccc install
-	make -C lib/isccfg install
-	make -C lib/bind9 install
-	make -C lib/irs install
-	make -C bin/dig install
-	make -C bin/delv install
-	make -C bin/nsupdate install
+termux_step_post_configure() {
+	# Android linker is unable to driectly reslove versoined libraries.
+	# This will create a symlink to versoined library via `libname.so`.
+	sed -i 's|library_names_spec=.*|library_names_spec="\\\$libname\\\$release\\\$shared_ext \\\$libname\\\$shared_ext"|g' ./libtool
```

Comment:
Hmm, why is this needed? The linux*android* case in libtool.m4 already contains exactly this spec:

```
linux*android*)
  version_type=none # Android doesn't support versioned libraries.
  need_lib_prefix=no
  need_version=no
  library_names_spec='$libname$release$shared_ext $libname$shared_ext'
```

is it matching and using another case?

Reply:
Weird....

I do not have (`$libname$shared_ext`) in `library_names_spec` locally.

Which version did you check in?